shared/chat/conversation/bot/install.tsx

```diff
@@ -77,7 +77,20 @@ const InstallBotPopup = (props: Props) => {
     if (!conversationIDKey) {
       return
     }
-    dispatch(Chat2Gen.createRemoveBotMember({conversationIDKey, username: botUsername}))
+    dispatch(
+      RouteTreeGen.createNavigateAppend({
+        path: [
+          {
+            props: {
+              botUsername,
+              conversationIDKey,
+              namespace: 'chat2',
+            },
+            selected: 'chatConfirmRemoveBot',
+          },
+        ],
+      })
+    )
```

Comment:
would be good to change this in `shared/teams/team/rows/bot-row/bot/container.tsx` too